Tutorial.playground/Contents.swift

```diff
@@ -0,0 +1,658 @@
+// Copyright (c) 2012-2014 Gabriel Gonzalez
```

Comment:
ZOMG, 💖💖

Just open an issue when you do.  I have a feeling we could wind up merging this real soon ;)